base/js/admin.js

```diff
@@ -542,7 +586,7 @@ var sowbForms = window.sowbForms || {};
 			});
 			var repeaterHtml = repeaterObject.html().replace(/_id_/g, $nextIndex);
 
-			var readonly = typeof $el.attr('readonly') != 'undefined';
+			var readonly = typeof $el.attr('readonly') !== 'undefined';
```

Comment:
I'm about 99% isEmpty doesn't check if it's undefined, only that it doesn't equal to 0. [isUndefined](http://underscorejs.org/#isUndefined) could very well be used though.